back/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const session = require('express-session');
+const { router } = require('./routing/index')
+const app = express();
+
+app.use(bodyParser.json());
+app.use(session({ secret: 'keyboard cat', cookie: { maxAge: 60000 } }))
```

Comment:
missing semicolons and please change secret key

Reply:
fixed